variables.mk

```diff
@@ -29,8 +29,7 @@ HELP_SIMULATION_VARIABLES = \
 "   LOADMEM                = riscv elf binary that should be loaded directly into simulated DRAM. LOADMEM=1 will load the BINARY elf" \
 "   LOADARCH               = path to a architectural checkpoint directory that should end in .loadarch/, for restoring from a checkpoint" \
 "   VERBOSE_FLAGS          = flags used when doing verbose simulation [$(VERBOSE_FLAGS)]" \
-"   timeout_cycles         = number of clock cycles before simulator times out, defaults to 10000000" \
-"   bmark_timeout_cycles   = number of clock cycles before benchmark simulator times out, defaults to 100000000"
```

Comment:
Isn't the `bmark` version used for RV tests?

Reply:
Nope, this isn't used anywhere in Chipyard, or in rocket-chip.